packages/commonwealth/client/scripts/helpers/index.ts

```diff
@@ -261,7 +261,7 @@ export function formatPercent(num: number, digits: number) {
 
 export function formatDuration(
   duration: moment.Duration,
-  includeSeconds = true
+  includeSeconds: boolean = true
```

Comment:
we dont need type here, even linter says that

Reply:
Addressed